maintainers/maintainer-list.nix

```
@@ -56,6 +56,12 @@
       fingerprint = "F466 A548 AD3F C1F1 8C88  4576 8702 7528 B006 D66D";
     }];
   };
+  "0x6d6178" = {
```

https://github.com/NixOS/nixpkgs/blob/e6ec21dfa3feaafccf90a5234d4f0b5c3a48cddb/maintainers/maintainer-list.nix#L29-L34

We now have a rule about handles that begin with a numeral.
```suggestion
  _0x6d6178 = {
```

 Please also rebase the PR on master.